openpype/plugins/publish/collect_otio_frame_ranges.py

```diff
@@ -27,6 +21,14 @@ class CollectOtioFrameRanges(pyblish.api.InstancePlugin):
     hosts = ["resolve", "hiero", "flame", "traypublisher"]
 
     def process(self, instance):
+        # Not all hosts can import these modules.
+        import opentimelineio as otio
```

Comment:
Are all the imports moved to `process` because of testing?

Reply:
That's right. It because of the pyblish error you get when loading a plugin file that has unavailable modules.

Have you got another solution in mind?

Reply:
We could potentially somehow disable the Pyblish errors, but I think its good practice to know which plugins are shared across multiple hosts and what issues that incurs.

Reply:
> Are all the imports moved to process because of testing?

Side note, there have been developers/studios reporting the warnings as well in Discord every now and then when e.g. publishing in certain hosts - thinking _they_ did something wrong.

Reply:
Makes sense.